Gemfile

```diff
@@ -28,6 +28,9 @@ gem 'ri_cal'
 gem 'rails_autolink'
 gem 'faker', require: false
 
+gem 'pry-rails'
+gem 'pry-byebug', require: false
```

Comment:
I wasn't thinking about where I would use byebug. It isn't necessary outside of dev and test. I will update it.